container-search/src/test/java/com/yahoo/search/test/QueryTestCase.java

```diff
@@ -68,6 +71,17 @@
  */
 public class QueryTestCase {
 
+    @Test
+    public void testIt() throws Exception {
+        JSONObject newroot = new JSONObject("{\"key\": 3}");
+        var hit = new FastHit();
+        hit.setField("data", (JsonProducer)s -> s.append(newroot));
+        var field = hit.getField("data");
+        if (field instanceof JsonProducer) {
+            System.out.println((((JsonProducer) field).toJson()));
+        }
+    }
+
```

Comment:
Leftover debugging from work with this, @bratseth?

Reply:
Idk, but looks removable yes.